src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/StandardRecipeManager.java

```diff
@@ -96,4 +111,20 @@ public void read(@NotNull final CompoundNBT compound)
         }
         cache = null;
     }
+
+    @Override
+    public void reset()
+    {
+        recipes.clear();
+        usedRecipes.clear();
+    }
+
+    @Override
+    public void registerUse(final IToken<?> token)
+    {
+        if(!usedRecipes.contains(token))
+        {
+            usedRecipes.add(token);
```

Comment:
No need to check contains - add already does that.

Reply:
ye it's a set, it shouldn't need contains

Reply:
Fair enough. Should be fixed.